server/parsers/basic_input_output_parser.ts

```diff
@@ -29,6 +29,14 @@ export const parseSuggestedActions = (value: string): string[] => {
     suggestedActions = [];
   }
 
+  /**
+   * The suggestedOutputString may be 'null'
+   * Fail fast here.
+   */
+  if (!suggestedActions) {
+    return [];
+  }
+
   if (suggestedActions.length) {
```

Comment:
Shall we check if `suggestedActions` was an array here? So we won't need to add `!suggestedActions` above. Just consider if `suggestedOutputString` would be a `number` or `boolean`, access `length` attribute would throw error.

Reply:
Actually, if suggestedActions is not an array, it will return [] in line 45.

Reply:
I got your point, let me change the block here to single `isStringArray`.